store/storetest/user_store.go

```diff
@@ -92,62 +92,57 @@ func testUserStoreSave(t *testing.T, ss store.Store) {
 		Username: model.NewId(),
 	}
 
-	if _, err := ss.User().Save(&u1); err != nil {
-		t.Fatal("couldn't save user", err)
-	}
+	_, err := ss.User().Save(&u1)
+	require.Nil(t, err, "couldn't save user")
```

Comment:
I this case I think the require makes sense, if I'm not able to create the initial user for the tests, I should stop testing.

Anyway is a good advice for other cases following this, for example most of the `NotNil` checks from the errors trying to save erroneous data can be `assert` instead of `require` because they can fail independently.